### PR TITLE
feat: [cr] adds a simple map with lat/long coordinates on probe edit page

### PR DESCRIPTION
**Pull request by jewbetcha**
Screen Shot 2022-01-19 at 1 41 32 PM

Preview for creating private probes as well as some visual cues on established public probes.

Addresses https://github.com/grafana/synthetic-monitoring-app/issues/84

### CODE REVIEW

File: src/components/SimpleMap/SimpleMap.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ComposableMap, Geographies, Geography, Marker } from 'react-simple-maps';
+
+const geoUrl = 'https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json';
```
Comment (rdubrock) [inline src/components/SimpleMap/SimpleMap.tsx:4]: make me slightly nervous to depend on an external resource like this. Is it reasonable to copy the json into our repo, or is that a bad idea?

**Review by rdubrock — APPROVED** [review]
🦉

File: src/components/SimpleMap/SimpleMap.tsx (hunk 1 above)
Comment (jewbetcha) [reply src/components/SimpleMap/SimpleMap.tsx:4]: Hmm yeah, let me see what I can do about that

Comment (electron0zero) [reply src/components/SimpleMap/SimpleMap.tsx]: have we looked into pulling in geomap panel? maybe we can use geomap with one layer and controls disabled 🤔

Comment (jewbetcha) [reply src/components/SimpleMap/SimpleMap.tsx]: We haven't, but in my past experience its not a trivial thing to use a panel outside of a dashboard

**Review by electron0zero — APPROVED** [review]

**Pull request merged**